epochStart/bootstrap/disabled/disabledPeerShardMapper.go

```diff
@@ -0,0 +1,30 @@
+package disabled
+
+import "github.com/ElrondNetwork/elrond-go-core/core"
+
+// peerShardMapper -
+type peerShardMapper struct {
+}
+
+// NewPeerShardMapper -
+func NewPeerShardMapper() *peerShardMapper {
+	return &peerShardMapper{}
+}
+
+func (p *peerShardMapper) GetPeerID(_ []byte) (*core.PeerID, bool) {
```

Comment:
missing & partial comments:
we should have something like 
"UpdatePeerIDPublicKeyPair does nothing" or
"GetPeerInfo returns a default instance" and such